packages/ui/src/lib/utils/index.ts

```diff
@@ -5,7 +5,7 @@ type JSONdecoded = string | number | boolean | object | Array<JSONdecoded>
 
 export type SavedSeedMessages = {
 	personaId: number
-	serializedMessages: string[]
+	serializedMessages: string[][]
 	idCommitment: string
 }
```

Comment:
This probably should not be here but if only used within zkitter then somewhere under `adapters/zkitter`